packages/next/src/utils/config.ts

```diff
@@ -88,27 +89,24 @@ export function createWebpackConfig(
 }
 
 export function prepareConfig(
-  workspaceRoot: string,
-  projectRoot: string,
-  outputPath: string,
-  fileReplacements: FileReplacement[],
   phase:
     | typeof PHASE_PRODUCTION_BUILD
     | typeof PHASE_EXPORT
     | typeof PHASE_DEVELOPMENT_SERVER
-    | typeof PHASE_PRODUCTION_SERVER
+    | typeof PHASE_PRODUCTION_SERVER,
```

Comment:
Just a thought - why don't we turn this into an enum?

Reply:
Not sure if there's an advantage to that? The consts are coming from next itself.